lobster/tools/codebeamer/codebeamer.py

```diff
@@ -332,14 +354,18 @@ def main():
         "base"       : "%s/cb/api/v3" % options.cb_root,
         "user"       : options.cb_user,
         "pass"       : options.cb_pass,
+        "token"        : options.cb_token,
         "verify_ssl" : not options.ignore_ssl_errors,
         "page_size"  : options.query_size,
         "timeout"    : options.timeout,
     }
 
     if options.config:
         if os.path.isfile(options.config):
-            cb_config["references"] = parse_cb_config(options.config)
+            cb_config["references"], config_token = (
+                parse_cb_config(options.config))
+            if not cb_config["token"] and config_token:
+                cb_config["token"] = config_token
```

Comment:
Simplify like this:
```python
if os.path.isfile(options.config):
    cb_config.update(parse_cb_config(options.config))
else:
    ap.error("cannot open config file '%s'" % options.config)
```
This implies that the config file always has precedence over the command line arguments.
That might be counter-intuitive. We can also do it the other way around:
```python
    cb_config = {}
    if options.config:
        if os.path.isfile(options.config):
            cb_config =parse_cb_config(options.config)
        else:
            ap.error("cannot open config file '%s'" % options.config)

    cb_config.update({
        "root"       : options.cb_root,
        "base"       : "%s/cb/api/v3" % options.cb_root,
        "user"       : options.cb_user,
        "pass"       : options.cb_pass,
        "token"      : options.cb_token,
        "verify_ssl" : not options.ignore_ssl_errors,
        "page_size"  : options.query_size,
        "timeout"    : options.timeout,
    })
```